orc8r/gateway/python/magma/magmad/generic_command/command_executor.py

```diff
@@ -40,7 +44,10 @@ async def execute_command(
         """
         Run the command from the dispatch table with params
         """
-        result = await self.get_command_dispatch()[command](params)
+        cmd = self.get_command_dispatch().get(command)
+        if not cmd:
+            raise CommandExecutionException(f"no config for {command}")
+        result = await cmd(params)
```

Comment:
Is there something we can do here to return a more helpful error if `params` are missing?

Reply:
I've added the above check as well as another in NMS via https://github.com/magma/magma/pull/14014/commits/c2590a39ce2d6f441f0977d9566db789d18661b3